src/android/AckDatabase.java

```diff
@@ -8,7 +8,7 @@
 import androidx.room.TypeConverters;
 import androidx.work.Data;
 
-@Database(entities = {UploadEvent.class, PendingUpload.class}, version = 4)
+@Database(entities = {UploadEvent.class}, version = 3)
```

Comment:
I am not sure here if you can downgrade the database directly like this

https://developer.android.com/training/data-storage/room/migrating-db-versions#handle-missing-migrations

We should use a fallback method

Reply:
if we keep the version at 4 it should be good right ? @zfir